src/dis6/SignalPdu.cpp

```diff
@@ -4,7 +4,8 @@ using namespace DIS;
 
 SignalPdu::SignalPdu()
     : RadioCommunicationsFamilyPdu(), _encodingScheme(0), _tdlType(0),
-      _sampleRate(0), _dataLength(0), _samples(0) {
+      _sampleRate(0), _dataLength(0), _samples(0)
+{
```

Comment:
@intothevoid can you undo all code formatting changes in the PR, such as this one that change the curly bracket location.

Reply:
Apologies that's just the auto linting that must have kicked in. I'll update the PR